README.md

```diff
@@ -1,2 +1,66 @@
-# Awesome-Books-App
-This is a solo version of a booklist app.
+## Awesome project
+
+> In this project, I made a simple website called Awesome Books. The user is able to add and remove books, the added books are arranged in a list at the top of the page, removing books is also possible with one click. I used different modules with javascript classes to make these functions possible. I used simple html to set up the pages. There is a list page, add new book page and contact page. 
+
+![](https://img.shields.io/badge/Microverse-blueviolet).
+
+<img src="https://github.com/elfin-git/Awesome-Books-App/blob/Feature/img/ui.png" width="600" height="500"/>
+
+## Built With
+
+- HTML & CSS
+- Github
+- Javascript
+
+## Live Demo 
+
+[liveDemoLink](https://elfin-git.github.io/Awesome-Books-App/)
+
+## Getting Started
+
+**To get a local copy up and running follow these simple example steps.**
+
+# Prerequisites
+
+- Github flow knowledge. -Intermediate CSS, HTML and Javascript. -Installation of linter files. 
+
+# Setup 
+- To clone the repository run [https://github.com/elfin-git/Awesome-Books-App] in your terminal
+
+# Usage
+- This can be used as a simple booklist app to add and remove your favourite books
+- **NOTE**: The mobile version is currently unavailable.
+
+# Install
+- Set up Github Actions
+- Set up linter > Lighthouse , Webhint ,Stylelint , Eslint.
+
+## Authors
```

Comment:
- [ ] OPTIONAL
Kindly change the header title _Authors_ to _Author_, as this project is done solely. This will make your ReadMe appear more professional.